internal/cloud/backup.go

```diff
@@ -16,4 +16,10 @@ type Backup struct {
 	// VaultName is the identifier of the place in the cloud where the archive was
 	// stored.
 	VaultName string
+
+	// Paths are the folders that were synchronized with this backup.
+	Paths []string
```

Comment:
This will be empty every time we synchronize the local database with the cloud, because the cloud doesn't known the paths. We could do a merge strategy, but not sure if it's the best idea.